wasm/src/diagnostic.rs

```diff
@@ -187,3 +187,27 @@ where
         message: labeled_span.label().map(ToString::to_string),
     }
 }
+
+pub fn interpret_errors_into_vs_diagnostics(
+    errs: &[interpret::Error],
+) -> Vec<(String, VSDiagnostic)> {
+    let default_uri: Arc<str> = Arc::from("<project>");
```

Comment:
Does this need to be `Arc<str>` or could it be `Rc<str>`? (I recall we had some Miette requirements that may come into play here)

Reply:
Yeah, `Arc<str>` did come from Miette, but actually I can avoid some unnecessary conversions by just declaring this as a string. Updating...